docs/install.rst

```diff
@@ -29,6 +29,11 @@ Astropy also depends on other packages for optional features:
 
 - `xmllint <http://www.xmlsoft.org/>`_: To validate VOTABLE XML files.
 
+- `matplotlib <http://matplotlib.org/>`_: To plot 2D data.
```

Comment:
Maybe change this from ``to plot 2D data`` to just something like ``To provide plotting functionality that `astropy.visualization` echances ``? It's not exactly 2D *data* that's being plotted, but rather *2D plots*.